Home_Work_2_task_2

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace L_2_array_task_2
+{
+    /*Преобразовать массив так, чтобы сначала шли все отрицательные элементы, а потом
+     * положительные (0 считать положительным)*/
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            const int size = 20;
+            int[] Masivkas = new int[size];
+            Fill_Masivkas(Masivkas, size);
+            Sort_Masivkas(Masivkas, size);
+            Ready_Sort_Masivkas(Masivkas, size);
+
+            //Random rand = new Random();
+
+        }
+
+        /// <summary>
+        /// Print our array with System.Rand from -20 to 20.
+        /// </summary>
+        /// <param name="Arr"></param>
+        /// <param size="s"></param>
```

Comment:
Describe input method value like

/// <param name="arr">Array</param>
/// <param name="size">Size of array</param>

Also you can use array.Length  instead of creating one more variable for size

local variable start  with a small letter

Reply:
Yes understand ,with array.Length it more elegant, without variable size;
Aga, local variable you are mean use "arr" instead "Arr"

Reply:
done it